extensions/amp-iframe/amp-iframe.md

```diff
@@ -92,6 +86,10 @@ The reasons for this policy are that:
 -   `amp-iframe` has no fully iframe controlled resize mechanism.
 -   Viewability information may not be available to `amp-iframe`.
 
+### Standalone use outside valid AMP documents
+
+Bento AMP allows you to use AMP components in non-AMP pages without needing to commit to fully valid AMP. You can take these components and place them in implementations with frameworks and CMSs that don't support AMP. Read more in our guide [Use AMP components in non-AMP pages](https://amp.dev/documentation/guides-and-tutorials/start/bento_guide/). To find the standalone version of `amp-iframe`, see [`bento-iframe`](./1.0/README.md).
```

Comment:
Until README.md exists, this will have a dead link issue in CI

Reply:
That's right. README.me is added in #36210 and will be merged soon.